Finish Basic Layout

Zibilyonik opened the pull request:
-Add basic layout using Bootstrap and CSS
-Implement Webpack
-Add linters
-Add descriptive README

Zibilyonik committed:
initialize branch

Zibilyonik committed:
add .gitignore, add webpack

Zibilyonik committed:
implement webpack

Zibilyonik committed:
add linters, restructure project for webpack

Zibilyonik committed:
test
-Double
-Triple'

Zibilyonik committed:
Add html content and styling
-Add css into project
-Add html elements
-Add bootstrap classes
-Change index.css filename to style.css

Zibilyonik committed:
Add linters and fix linter issues

Zibilyonik committed:
Update README

carlos-ssh commented on src/index.html:
```diff
@@ -0,0 +1,56 @@
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <title>THE BEST OF THE BEST</title>
+  </head>
+  <body>
```
- Index.html main contain only root element

Zibilyonik replied:
Hello, can you show me at what part of the project description is this listed as a requirement? I've looked at the page and there was no mention of anything except basic HTML markup.
![image](https://user-images.githubusercontent.com/34018257/127254776-464d46f4-97b7-48ef-9c35-64e7ff2d216f.png)

carlos-ssh replied:
In your last point:
![Screen Shot 2021-07-27 at 21 38 54](https://user-images.githubusercontent.com/22258274/127254909-66cf6179-2afc-4d68-82ee-9f8d850aa2a8.png)

Zibilyonik replied:
With this logic, I should also create the entire HTML file inside JS. I don't understand your point, the project IS using ES6, and it's never written that the HTML can ONLY contain the root element.

Zibilyonik replied:
As you can see, Week 2's project clearly states that it should be created dynamically. There is no such requirement listed in this project.
![image](https://user-images.githubusercontent.com/34018257/127256905-51b6d4ca-0851-4785-940b-5fb7c03a9872.png)

carlos-ssh commented on src/index.js:
```diff
@@ -0,0 +1 @@
+import './style.css';
```
**Use ES6 modules to write modular JavaScript with proper ES6 syntax.**


- usage of `` arrow functions ``  instead of the `` function `` keyword.
- usage of `` import `` and `` export``  instead of require or `` module.exports``.
- proper usage of `` const`` and ``let`` keywords.

carlos-ssh reviewed (changes requested):
Hi @Zibilyonik, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Your project it's really cool but are needed to do a few changes.  
![Screen Shot 2021-07-27 at 21 32 16](https://user-images.githubusercontent.com/22258274/127254366-1fdf82b3-6800-4f55-bd69-70bcee2c0dec.png)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Zibilyonik committed:
Change code to fill HTML by JS
-Clean out index.html
-Create new markup.js module
-Add code to call markup.js inside index.js

Zibilyonik committed:
Build project with webpack

Zibilyonik committed:
Fix linter config

Zibilyonik committed:
Revert implementing of requested changes

This reverts commit 1d5795abfd48182c34ce71f2428eeb74ff5b5eeb.

Zibilyonik committed:
Revert "Build project with webpack"

This reverts commit 316d0b3b4077bc00b92f25ed6359c6e554f71a5d.

Zibilyonik committed:
Revert "Change code to fill HTML by JS"

This reverts commit 3d8b0852c09f0fbaddaa4bd09500669bb42536b9.

mimipeshy reviewed (approved):
## Hi @Zibilyonik 👋🏾 👋🏾 , 

First of all, you did a great job on this project 😄 We understand there is a conflict in the project requirements that you have been provided with. We truly apologize for all inconveniences caused and we are working hard to resolve the issue so it does not happen again. 

Therefore,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._
